### ISSUE TITLE
Compiler crash on empty args

### ISSUE THREAD

**Issue by yhara** [shiika-lang/shiika#505]
## Given
`[1].push`
## Expected
Some error (like `push takes 1 argument`) shown
## Actual
```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', lib/skc_ast2hir/src/convert_exprs/method_call.rs:109:26
```

**Comment by EruEri** [comment]
Hi @yhara 

could we use the location of the method if args.locs is None ? 

So for this example 
```crystal
let a = Array.build<Int>(17) do |x: Int|
    x + 1
end

a.push
```

The error message would be 
```
Error: Error: missing argument `item' of method `Array#push(item: Int) -> Void'
   ╭─[main.sk:5:2]
   │
 5 │ a.push
   ·  ──┬──  
   ·    ╰──── missing argument `item' of method `Array#push(item: Int) -> Void'
───╯


```

**Comment by yhara** [comment]
@EruEri Sounds good!

**Comment by yhara** [comment]
fixed by #516.
